nservicebus/recoverability/configure-delayed-retries.md

```diff
@@ -28,24 +29,21 @@ snippet:DisableDelayedRetries
 
 ## Custom Retry Policy
 
-Custom retry logic can be configured based on headers or timing in code.
-
-
-### Applying a custom policy
+Custom retry logic can be configured via code.
 
 snippet:DelayedRetriesCustomPolicy
 
 
 ### Simple Policy
 
-The following retry policy that will retry a message 3 times with a 5 second interval.
+The following retry policy overrides default delay interval and sets it to 5 seconds.
```

Comment:
>The following retry policy overrides default delay interval and sets it to 5 seconds.

What's the default delay?  Can we use something other than 5 if 5 is the default?